apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -27,7 +27,7 @@ describe('[Chat]', function () {
 				.expect(400)
 				.expect((res) => {
 					expect(res.body).to.have.property('success', false);
-					expect(res.body).to.have.property('error', '[invalid-channel]');
+					expect(res.body).to.have.property('errorType', 'invalid-params');
```

Comment:
we should not change error types/messages

Reply:
I agree with not changing the properties, but what about the messages? Wouldn't it be better to keep the properties as it was, but also maintain the validations for the endpoints?

Reply:
I've rolled everything back 🤙